modules/webp-default/load.php

```diff
@@ -0,0 +1,20 @@
+<?php
+/**
+ * WebP-Default Module
+ *
+ * @package performance-lab
+ * @since 1.0.0
+ */
+
+/**
+ * Module Name: WebP default.
+ * Description: Use WebP as the default sub sized image format.
+ *              This module changes the default format WordPress uses for
+ *              generating sub sized images from JPEG to WebP. Requires that the
+ *              server supports WebP and only acts on JPEG image uploads.
+ * Focus: images
+ * Experimental: No
+ */
```

Comment:
Maybe stick to the simple module header like it is in https://github.com/WordPress/performance/blob/trunk/docs/Writing-a-module.md#example? It isn't necessary to have to separate header comments here.

Also, is it possible to shorten the description? There's no hard limit, but I think we should stick to the plugin convention of a brief 1-sentence description in third person (e.g. `Uses WebP as the default format for new JPEG image uploads.`). If we require more space for longer readme- or tutorial-like content, we may need to expand the module specification, but that should then rather happen in a separate module-independent issue.

Reply:
When I tried adding just the module header, phpcs linter complained about a "Missing file doc comment". I could silence the warning? Adding the block here was the only way I found to satisfy the linter.

👍🏼  - will shorten the description since this text may get surfaced somewhere.

Reply:
simplified header